src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Views/MainPage.xaml.cs

```diff
@@ -15,16 +15,15 @@
 
 // The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkId=402352&clcid=0x409
 
-namespace $ext_safeprojectname$
+namespace $ext_safeprojectname$;
```

Comment:
This change is quite scary. Do we expect users to be on C# 10?

Reply:
I see now `AppHead.xaml.cs` is using file-scoped namespaces, so probably the change is okay.

Reply:
Yes, in this specific template it should be ok - it targets either .NET 6 or 7, so C# 10+